XML/VMConfigurations/OneVM.xml

```diff
@@ -17,6 +17,24 @@
             </VirtualMachine>
         </ResourceGroup>
     </OneVM>
+    <OneVMhb>
+        <isDeployed>NO</isDeployed>
```

Comment:
no need add a new type, use OneVM and add OverrideVMSize in test case definition, and here it already attached the data disk, you don't need add again in the ps1 file.

Reply:
I will play with them first.

Reply:
OneVM, D1_v2 does not provide enough CPU power, so caught 100% time-out during git clone, not even kernel compilation. Like your comment, if we change to OneVM a.k.a. D1_v2, then the execution should take OverrideVMSize parameter all the times. If we keep OneVMhb, D4_v2 then we do not need extra parameter. Comment back to me, @LiliDeng